devito/dse/rewriters.py

```diff
@@ -408,5 +418,22 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
 class CustomRewriter(AggressiveRewriter):
 
+    passes_mapper = {
+        'new_pass': AggressiveRewriter._new_pass
```

Comment:
we need the actual passes here, with reasonable names. I suggest you propose them ?

Reply:
ok, answered before. Let's find a name, I think it will be temporary as soon as skewing is introduced.

Reply:
no I think we need names for the current passes, such as `_extract_sum_of_products`, `_extract_time_invariants`, etc

sorry I hadn't explained myself properly before